diesel/src/associations/mod.rs

```diff
@@ -389,25 +386,8 @@ impl<'a, T: HasTable> HasTable for &'a T {
 /// `update(YourStruct::table().find(&your_struct.primary_key())`).
 ///
 /// This trait is usually implemented on a reference to a struct,
-/// not the struct itself.
-///
-/// ### Deriving
+/// not the struct itself. It can be [derived](derive.Identifiable.html).
```

Comment:
`not on the struct itself` ?

Reply:
What should I change that to? But I believe that is for another issue, maybe not in this pull request.

Reply:
Ahh you are correct, it had this exact language before, I will create a PR after this is merged.

For context I think
`This trait is usually implemented on a reference to a struct, not the struct itself.`
should be
`This trait is usually implemented on a reference to a struct, not on the struct itself.`

Reply:
Sorry, @martell I just noticed that my context was also wrong, I thought I was in another repository and another pull request, it also have a similar name.

Do we need the `usually`? Or should we have an example for this?